docs/BR.md

```diff
@@ -1598,9 +1591,7 @@ b. `cRLDistributionPoints`
 
 c. `authorityInformationAccess`
 
-   With the exception of stapling, which is noted below, this extension MUST be present. It MUST NOT be marked critical, and it MUST contain the HTTP URL of the Issuing CA's OCSP responder (`accessMethod` = 1.3.6.1.5.5.7.48.1). It SHOULD also contain the HTTP URL of the Issuing CA's certificate (`accessMethod` = 1.3.6.1.5.5.7.48.2).
-
-   The HTTP URL of the Issuing CA's OCSP responder MAY be omitted, provided that the Subscriber "staples" the OCSP response for the Certificate in its TLS handshakes [RFC4366].
+   This extension MUST be present. It MUST NOT be marked critical, and it MUST contain the HTTP URL of the Issuing CA's OCSP responder (`accessMethod` = 1.3.6.1.5.5.7.48.1). It SHOULD also contain the HTTP URL of the Issuing CA's certificate (`accessMethod` = 1.3.6.1.5.5.7.48.2).
```

Comment:
Why is this extension a mandatory requirement when it was previously allowed not to include if stapling was enforced? It seems that the current Mozilla Root Store Policy doesn't require it also (under Section 6. Revocation):

> For end-entity certificates, if the CA provides revocation information via an Online Certificate Status Protocol (OCSP) service:

Reply:
Could you help explain what you found confusing in the issue description, where this is addressed?

It currently reads:
> The Microsoft Root Program expresses a more tightly defined certificate profile for OCSP, and also places requirements on CRLs for intermediates. These requirements include:
>
> * OCSP MUST be supported, with no exceptions. Previously, an exception was given to contractually-defined OCSP stapling, allowing the responder URL to be omitted from the certificate. (Section 7.1.2.3 (c))

Reply:
The existing Microsoft program states:
> All end-entity certificates must contain an AIA extension with a valid OCSP URL. These certificates may also contain a CDP extension that contains a valid CRL URL. All other certificate types must contain either an AIA extension with an OCSP URL or a CDP extension with a valid CRL URL

Reply:
I was under the impression that the alignment ballot wanted to capture the common policies between Root Programs, which is why I highlighted this issue. HARICA is not affected by this change but I thought that it was a good idea to exclude this requirement for Subscribers with high-volume traffic that could potentially cause problems to the CA's OCSP responders.

Reply:
Ah, no, you've misunderstood the purpose of this ballot. I'm definitely open for suggestions on how to make that clearer, but I tried very hard in the introduction to address this.

These are all the requirements that Root Programs impose that are not in the BRs, and which other Root Programs did not raise concerns over. The only exception I'm aware of is P-521, which is permitted by Microsoft (and they would like to continue as such) but forbidden by Mozilla and Google (and they would like it to continue as such). Beyond that, any requirement that a *single* root program has is being proposed here, because CAs are effectively already subject to this requirement, and it improves the status quo by adding to the BRs and making this auditable.

However, at least by raising this issue, you highlighted an opportunity to clarify here, which is Microsoft policy only requires one of OCSP and/or CRLs. I'll correct that.